google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/integration/AppEngineServerHelper.java

```diff
@@ -44,6 +50,28 @@ public ApplicationServerPersistentData createPersistentDataEmptyInstance() {
   }
 
   public ApplicationServerPersistentDataEditor createConfigurable() {
-    return new AppEngineServerEditor();
+    final CloudSdkPanel cloudSdkPanel = new CloudSdkPanel();
+
+    return new ApplicationServerPersistentDataEditor<ApplicationServerPersistentData>() {
+      @Override
+      protected void resetEditorFrom(ApplicationServerPersistentData s) {
```

Comment:
Maybe rename 's' to a more meaningful name? like `data` or `persistedConfiguration` (which would make the applyEditorTo arg be called something like `toBePersistedConfiguration`.